Final_project_2/frontend/index.html

```diff
@@ -347,49 +346,43 @@ <h6>We love talking and hearing from you</h6>
                     </div>
                 </div>
             </div>
-        <div>
-            <address class="information">
-            <span>
+        <div class="address-block">
+            <address class="address-item">
              <i class="fa fa-envelope"></i>
-             info@theband.com
```

Comment:
It is quite uncommon to have text like this just in HTML. Usually all text is immediately wrapped up in a `div`, `span` etc (I've opted for `span` in this case). This means that it can be styled by CSS - because it is directly in an HTML tag, then the browser can apply styles to it, such as the centering styles.